src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, operand) {
+      this.result = callback(operand, this.result);
+
+      return this;
```

Comment:
The `operate` method should update the `result` property by applying the operation to the current `result` value and the provided operand. The current implementation incorrectly swaps the order of `operand` and `this.result` when calling the callback.

Reply:
Consider fixing the AI comment